### PR TITLE
Proofread

### PR DESCRIPTION
**Pull request by OdelGreg**

### CODE REVIEW

File: manuals/1.0/en/database.md [hunk 1]
```diff
@@ -221,16 +596,39 @@ class Index
...
+ */
+public function setLogDb(Connection $logDb)
+```
+
 [MasterSlaveConnection](http://www.doctrine-project.org/api/dbal/2.0/class-Doctrine.DBAL.Connections.MasterSlaveConnection.html) is provided for master/slave connections.
 
 # CakeDb
 
-**CakeDb** is the database access module for the CakePHP3 Database library. This module is provided by [@lorenzo](https://github.com/lorenzo) ( original author of CakeDb).
+**CakeDb** is an ORM using the active record and data mapper pattern idea. It is the same as the one provided in CakePHP3.
```
Comment (koriym) [inline manuals/1.0/en/database.md]: the credit (lorenzo ) should not be removed.

File: manuals/1.0/en/database.md [hunk 2]
```diff
@@ -180,6 +212,349 @@ use Ray\AuraSqlModule\Annotation\Transactional;
     }
 ```
 
+To do transactions on multiple connected databases, specify properties in the `@ Transactional` annotation.
```
Comment (koriym) [inline manuals/1.0/en/database.md]: should be `@Transactional`

File: manuals/1.0/en/database.md [hunk 3]
```diff
@@ -180,6 +212,349 @@ use Ray\AuraSqlModule\Annotation\Transactional;
...
+        $sth->execute($this->insert->getBindValues());
+        // or
+        // $sth = $this->pdo->perform($this->insert->getStatement(), this->insert->getBindValues());
+
+        // get the last insert ID
+        $name = $insert->getLastInsertIdName('id');
+        $id = $pdo->lastInsertId($name);
+```
+
+The `cols()` method can also pass an associative array with the column name as a key and the value is a bind value.
```
Comment (koriym) [inline manuals/1.0/en/database.md]: better to use original text ?

File: manuals/1.0/en/database.md (hunk 1 above)
Comment (OdelGreg) [inline manuals/1.0/en/database.md]: The credit Lorenzo is in the lower part.

Comment (OdelGreg) [inline manuals/1.0/en/database.md]: I updated the CakeDb part based on what is on the Japanese version https://bearsunday.github.io/manuals/1.0/ja/database

File: manuals/1.0/en/database.md (hunk 3 above)
Comment (OdelGreg) [inline manuals/1.0/en/database.md]: Sorry, what do you mean here?

File: manuals/1.0/en/database.md (hunk 2 above)
Comment (OdelGreg) [inline manuals/1.0/en/database.md]: I will update.

File: manuals/1.0/en/database.md (hunk 3 above)
Comment (koriym) [inline manuals/1.0/en/database.md]: Aura.SqlQuery has same explanation ?

File: manuals/1.0/en/database.md (hunk 1 above)
Comment (koriym) [inline manuals/1.0/en/database.md]: I see. Thanks.

File: manuals/1.0/en/database.md (hunk 3 above)
Comment (OdelGreg) [inline manuals/1.0/en/database.md]: I got it (from their site):

The cols() method allows you to pass an array of key-value pairs where the key is the column name and the value is a bind value (not a raw value)

Comment (koriym) [inline manuals/1.0/en/database.md]: ok thanks.

File: manuals/1.0/en/database.md (hunk 2 above)
Comment (OdelGreg) [inline manuals/1.0/en/database.md]: Updated.

File: manuals/1.0/en/database.md (hunk 3 above)
Comment (OdelGreg) [inline manuals/1.0/en/database.md]: Updated.

**Pull request merged**